Source/WebKit/UIProcess/API/Cocoa/WebKitSwiftOverlay.swift

```diff
@@ -23,8 +23,11 @@
  * THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-@_exported import WebKit
+#if !os(tvOS) && !os(watchOS)
+
+#if USE_APPLE_INTERNAL_SDK
```

Comment:
Just to confirm, is it OK to use WTF macros here?

Reply:
WTF macros don't work, but this is defined in CommonBase.xcconfig: https://github.com/WebKit/WebKit/blob/f55d6cc96b847b1a4c5b1ed4d5405d14dd601126/Configurations/CommonBase.xcconfig#L52-L54